samples/projects/java-project/src/main/java/example/Runner.java

```diff
@@ -0,0 +1,21 @@
+package example;
+
+import horseshoe.*;
+import java.io.*;
+import java.nio.file.*;
+import java.util.*;
+
+public class Runner {
+
+	static final Path outputFile = Paths.get("build/generated/HelloWorld.txt");
```

Comment:
I'm fine if we shove the output to somewhere other than the results. But my original intention was that any changes in the output from the template would show up as an unstaged file, requiring the user to commit the file and acknowledge that they either changed the sample template in some way to change the output, or they modified the library in such a way that existing templates would generate different output (which would be bad).

It probably should be enforced somewhere in the review process that no unstaged files should exist after running tests. If you have any ideas on how to do this, let me know.